core/auth/src/main/java/software/amazon/awssdk/auth/credentials/ProcessCredentialsProvider.java

```diff
@@ -111,6 +100,26 @@ private ProcessCredentialsProvider(Builder builder) {
         this.processCredentialCache = cacheBuilder.build();
     }
 
+    private List<String> executableCommand(Builder builder) {
```

Comment:
This means that if someone, with say a command that needed no args, migrates from
```
            ProcessCredentialsProvider.builder().command("script")).build()
```
to
```
            ProcessCredentialsProvider.builder().command(Arrays.asList("script"))).build()
```
because they see the deprecation warning on the previous method, there will be a behavior change where it used to be run in a shell and now it won't? I'm not too sure, so just asking to double-check if there are risks with that behavior change?

Reply:
Yeah thats correct, that is the intent of the change, in order to remove risk of command injections. I don't think there is a risk with changing to not-in-a-shell. I've added [description in the javadocs for the deprecated method](https://github.com/aws/aws-sdk-java-v2/blob/hdavidh/process-creds-list-of-strings/core/auth/src/main/java/software/amazon/awssdk/auth/credentials/ProcessCredentialsProvider.java#L288)  so users should be aware, they can continue to use it if they need the command executed in a shell